src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -114,7 +116,11 @@ export default function RequestedAppointmentDetailsPage() {
     );
   }
 
-  if (!appointment || appointmentDetailsStatus === FETCH_STATUS.loading) {
+  if (
+    !appointment ||
+    appointmentDetailsStatus === FETCH_STATUS.loading ||
+    (useV2 && !providerData)
```

Comment:
```suggestion
   // (useV2 && !providerData)
```
When a user does not select a provider, it will perpetually show the message, "Loading your appointment request...". You can remove the above condition to bypass the message.